### PR TITLE
FIX: getPassphrase crash

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Review by limpbrains — APPROVED** [review]

File: screen/wallets/export.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ const WalletExport = () => {
             )}
           </React.Fragment>
         ))}
-        {wallet.getPassphrase() && (
+        {wallet.getPassphrase && wallet.getPassphrase() && (
```
Comment (Overtorment) [inline screen/wallets/export.js:139]: Hmm. Maybe add this to abstractwallet ? Cc @limpbrains

**Review by Overtorment — APPROVED** [review]

File: screen/wallets/export.js (hunk 1 above)
Comment (limpbrains) [reply screen/wallets/export.js:139]: We agreed not to show passphrase on export wallet screen. This whole section will be removed on my next PR

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gpby0fmgc1gab1hwcqtm5aqekr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7322)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**